target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -375,6 +376,11 @@ ramips_setup_macs()
 	linksys,e1700)
 		wan_mac=$(mtd_get_mac_ascii config WAN_MAC_ADDR)
 		;;
+	snr,cpe-w4n-mt)
+		lan_mac=$(mtd_get_mac_binary Factory 0x28)
+		wan_mac=$(mtd_get_mac_binary Factory 0x2e)
+		label_mac=$wan_mac
```

Comment:
Please use nvmem to assign the mac addresses to the interfaces.

For example like this:
```
&factory {
	compatible = "nvmem-cells";
	#address-cells = <1>;
	#size-cells = <1>;

	macaddr_factory_28: macaddr@28 {
		reg = <0x28 0x6>;
	};

	macaddr_factory_2e: macaddr@2e {
		reg = <0x2e 0x6>;
	};
};
````

Reply:
Sorry I think this does not work, because the device has a switch and the switch is not using DSA yet. This would only make sense if you also use the `macaddr_factory_28` nodes.

Reply:
Remove changes or leave as is now?

Reply:
please remove them